sdc/tests/test_series.py

```diff
@@ -1295,6 +1295,65 @@ def test_impl(A):
         pd.testing.assert_series_equal(
             hpat_func(S), test_impl(S))
 
+    @skip_sdc_jit('Not impl in old style')
+    def test_series_slice_nonidx_nostart(self):
+        def test_impl(A):
+            return A.loc[-4:301]
+        hpat_func = self.jit(test_impl)
+
+        S = pd.Series([2, 4, 6, 6, 3], [-22, -5, -2, 300, 40000])
+        pd.testing.assert_series_equal(hpat_func(S), test_impl(S))
+
+
+    @skip_sdc_jit('Not impl in old style')
+    def test_series_slice(self):
+        def test_impl(A):
+            return A.loc[1:5]
+        hpat_func = self.jit(test_impl)
+
+        S = pd.Series([2, 4, 6, 6, 3], [1, 3, 5, 13, 22])
+        pd.testing.assert_series_equal(hpat_func(S), test_impl(S))
```

Comment:
Looks like test `test_series_slice_loc_start_stop` covers the same functionality as tests `test_series_slice_nonidx_nostart` and `test_series_slice`. So I propose to remove excess tests to not load CI.